### ISSUE TITLE
Webcomic theme not working with Inkblot's appearance customization menu

### ISSUE THREAD

**Issue by angelramoz** [mgsisk/webcomic#223]
Hello, i have noticed that the Webcomic theme is not compatible with Wordpress 3.83+ in the plugin's main download page so this may be the main reason behind my problem right now. I can only download Wordpress' newest 4.1 version which makes it impossible to use this plugin to its fullest.

Everything seems to be working correctly, but the plugin only seems to not work when trying to customize the appearance with the Inkblot theme.  Is there any way around this in the coding?

Or if anyone knows- is there possibly a way for me to downgrade Wordpress?

**Comment by Mihari** [comment]
Download the appropriate WordPress from here: https://wordpress.org/download/release-archive/

1. Download WP 3.9.2
2. Delete the wp-config.php file within that package if it exists.
3. Remove the /wp-includes folder and its content.
4. Simply upload all the files and replace the files on your server.
5. Go to login page. You will see a notice about upgrading your Database. Just click the button.

**Comment by mgsisk** [comment]
Thanks for the downgrade instructions, Mihari, and for understanding the delay, and apologies to both of you for the trouble. This has been a known issue since WordPress 4, actually, but I took too long working on updating Inkblot for WordPress and found out that some of my updates were having issues in WordPress 4.1!

The good news is that I've corrected those and and working to get an update on GitHub by this weekend. It'll take a bit longer to show up on the official Theme Directory, as it has to go through theme review (which may require multiple rounds of revisions).
